src/ParseServer.js

```diff
@@ -323,6 +329,25 @@ class ParseServer {
     return parseServer.start(options, callback);
   }
 
+  /**
+   * @static
+   * Create an async parse server
+   * @param {ParseServerOptions} options the parse server initialization options */
```

Comment:
We should add comment linting finally, in the meantime could you move the `*/` into the next line?